### PR TITLE
[12.0][MIG] stock_change_price_at_date

### PR DESCRIPTION
**Pull request by kittiu**
WIP (migrate from v10->v12)

### CODE REVIEW

**Comment by rousseldenis** [comment]
@kittiu Could you fix this ?

**Comment by kittiu** [comment]
> @kittiu Could you fix this ?

Yes. Was waiting to be sure that this module worth doing. Will fix it shortly.

File: stock_change_price_at_date/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+    'name': 'Stock Change Price At Date',
+    'summary': """
+        This module allows to fill in a date in the standard wizard that
+        changes product price. That helps accountant to add some product
+        past values""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu',
```
Comment (nikul-serpentcs) [inline stock_change_price_at_date/__manifest__.py]: https://github.com/OCA/stock-logistics-workflow

File: stock_change_price_at_date/models/product_product.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
```
Comment (nikul-serpentcs) [inline stock_change_price_at_date/models/product_product.py]: Not Needed, remove Please

File: stock_change_price_at_date/tests/test_change_price_at_date.py [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+        self.wizard_obj = self.env['stock.change.standard.price']
+        self.move_line_obj = self.env['account.move.line']
+        self.product = self.env.ref('stock.product_icecream')
+        self.product.categ_id.property_stock_journal = self.env[
+            'account.journal'].create({
+                'name': 'Stock journal',
+                'type': 'sale',
+                'code': 'STK00'})
+        # Defining accounts
+        self.account = self.env['account.account'].create({
```
Comment (nikul-serpentcs) [inline stock_change_price_at_date/tests/test_change_price_at_date.py]: `self.env['account.account']` use multiple time, So use like global.

**Review by nikul-serpentcs — COMMENTED** [review]
nitpicking

File: stock_change_price_at_date/__manifest__.py (hunk 1 above)
Comment (kittiu) [reply stock_change_price_at_date/__manifest__.py]: Thanks!

**Comment by kittiu** [comment]
@nikul-serpentcs all comment fixed.
@rousseldenis done migration

**Review by HaraldPanten — APPROVED** [review]
Functional review. LGTM 👍

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Comment by kittiu** [comment]
This can be merged?

**Pull request merged**